arcsi/templates/show/add.html

```diff
@@ -44,6 +44,11 @@ <h2>{% block title %}Add radio show to archive{% endblock %}</h2>
       </select>
   </div>
 
+  <div>
+    <label for="taglist">Show tag list (separate with commas)</label>
```

Comment:
We should add these modifications for the episodes/items in the future.

Reply:
Sure, good idea why not have them there!